src/api/procedures/consumeAddMultiSigSignerAuthorization.ts

```diff
@@ -102,24 +92,24 @@ export async function getAuthorization(
 
   let calledByTarget: boolean;
 
+  let permissions;
   if (target instanceof Account) {
     calledByTarget = currentAccount.address === target.address;
     hasRoles = calledByTarget;
-    transactions = [TxTags.multiSig.AcceptMultisigSignerAsKey];
+    // An account accepting multisig cannot be part of an Identity, so we cannot check for permissions
   } else {
     calledByTarget = !!identity?.isEqual(target);
     hasRoles = calledByTarget;
     transactions = [TxTags.multiSig.AcceptMultisigSignerAsIdentity];
```

Comment:
No need for `transactions` to be defined in the outer scope if it's only used here